server/src/database/models/userAttainableGrade.ts

```diff
@@ -7,22 +7,22 @@ import {
 } from 'sequelize';
 
 import { sequelize } from '..';
-import CourseAssignment from './courseAssignment';
+import Attainable from './attainable';
 import User from './user';
 
-export default class UserAssignmentGrade extends Model<
-  InferAttributes<UserAssignmentGrade>,
-  InferCreationAttributes<UserAssignmentGrade>
+export default class UserAttainableGrade extends Model<
```

Comment:
"Study attainment" was chosen as the term to replace "assignment," but the reason we thought of using the name `Attainable` instead of `Attainment` in the backend to represent what we previously called "assignments" is because "attainment" gives an impression that something has already been attained. But in context, the "study attainments" aren't something that have been attained, they are something to be attained, so the term "attainable" was suggested.

But this model does represent something that has already been attained, so `AttainmentGrade` would be an appropriate name, or maybe `AttainedGrade`.

Reply:
renamed the model to `UserAttainmentGrade`, kept user in the name to emphasize this is user related.